core/controllers/contributor_dashboard_test.py

```diff
@@ -1796,6 +1796,157 @@ def test_get_translatable_topic_names(self) -> None:
         )
 
 
+class TranslatableTopicNamesPerClassRoomHandlerTest(test_utils.GenericTestBase):
+    """Test for the TranslatableTopicNamesByClassRoomHandlerTest."""
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.signup(self.CURRICULUM_ADMIN_EMAIL, self.CURRICULUM_ADMIN_USERNAME)
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+
+        self.admin_id = self.get_user_id_from_email(self.CURRICULUM_ADMIN_EMAIL)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+
+        self.set_curriculum_admins([self.CURRICULUM_ADMIN_USERNAME])
+
+    def test_get_translatable_topic_names_per_classroom(self) -> None:
```

Comment:
Try to break this test into small tests, such that each test, tests a single behavior